File: testprojects/java/accuracy/domain/indirect/allowedfrom/AccessStaticVariableIndirect_MethodVar.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessStaticVariableIndirect_MethodVar {
	private BackgroundService bckg;
	private String test;

	public AccessStaticVariableIndirect_MethodVar(){
		bckg = new BackgroundService();
	}
		//  Indirect access of a static attribute
		@SuppressWarnings("static-access")
		public String case5() {
			test = bckg.getServiceOne().sName;
			return test;
		}
}

File: testresources/csharp/benchmark/Domain/Indirect/ViolatingFrom/AccessStaticVariableIndirect_MethodVar.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class AccessStaticVariableIndirect_MethodVar
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

        private string test;

        public void Case5()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
            test = bckg.GetServiceOne().sName;
        }
	}
}
